client/app/startup/clientRegistration.js

```diff
@@ -14,3 +16,8 @@ ReactOnRails.register({
 
 ReactOnRails.registerStore({
 });
+
+if (window) {
```

Comment:
Does this really work if `window` is not defined? My hunch is that this will throw an error "window is not defined" or something like that.

To my understanding, the right way to check this is:

```
if(typeof window !== "undefined") {
  ...
}
```